docs/user_guide/sd_fingertips.rst

```diff
@@ -1,6 +1,100 @@
 Fingertips
 ============
 
+STF Sensor
+----------
+The Shadow Tactile Fingertip (STF) sensors are magnetic-based tactile sensors fitted in the
+standard fingertips or (slightly bigger) thumb shapes.
+
+
+.. figure:: ../img/sd_stf_sensors.png
+    :width: 40%
+
+It leverages 17 magnetic sensors evenly distributed at its core. For each sensor, a magnet
```

Comment:
The word "leverages" feels very clumsy. "uses"?  "comprises"? "is made from"?

Reply:
Thanks! Addressed in https://github.com/shadow-robot/sr_dexterous_hand_documentation/commit/66bb6dfef57d154b72092a475a02c1296d76c375